src/main/java/mate/controller/drivers/GetAllDriversController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller.drivers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/all")
+public class GetAllDriversController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService manufacturerService = (DriverService) injector
```

Comment:
The variable name 'manufacturerService' is not appropriate for its type 'DriverService'. Rename it to 'driverService' for better readability.